packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -119,11 +119,6 @@ class Config(_BaseConfig):
                 {"name": "env", "type": "string", "value": ["DISPLAY=:0"]},
                 # SEE 'simcore.service.settings' label annotations for simcore/services/dynamic/jupyter-octave-python-math:1.6.5
                 {"name": "ports", "type": "int", "value": 8888},
-                {
-                    "name": "constraints",
-                    "type": "string",
-                    "value": ["node.platform.os == linux"],
-                },
                 {
```

Comment:
Why did you remove this example?
```json
{
  "name": "constraints",
  "type": "string",
  "value": ["node.platform.os == linux"],
}
```

If it is deprecated, then you need to add a validator to convert from the old to the new version.
Take into account that this model validates against old service labels (remember that the registry is full of unreliable labels!)
See as example ``ensure_backwards_compatible_setting_type`` that I had to add in https://github.com/ITISFoundation/osparc-simcore/pull/3041 to fix issue with old jupyter labels

Reply:
it's a duplicate from another example